### PR TITLE
Fix image comparison in fragmentoutput test

### PR DESCRIPTION
**Pull request by qkmiao**
Most tests in fragmentoutput.html can pass now except some tests related to sRGB. I will fix sRGB cases next time.

### CODE REVIEW

File: sdk/tests/deqp/framework/common/tcuImageCompare.js [hunk 1]
```diff
@@ -279,8 +281,8 @@ tcuImageCompare.floatUlpThresholdCompare = function(imageSetName, imageSetDesc,
 
                 // Instead of memcpy(), which is the way to do float->uint32 reinterpretation in C++
                 for (var i = 0; i < refPix.length; i++) {
-                    refBits[i] = refPix[i];
-                    cmpBits[i] = cmpPix[i];
+                    refBits[i] = tcuFloat.convertFloat32Inline(refPix[i], tcuFloat.description32);
+                    cmpBits[i] = tcuFloat.convertFloat32Inline(cmpPix[i], tcuFloat.description32);
```
Comment (qkmiao) [inline sdk/tests/deqp/framework/common/tcuImageCompare.js]: Previous method cann't handle comparison between -1 and -0.9999999. Result of tcuFloat.convertFloat32Inline can be used to do float number comparison.

File: sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js [hunk 2]
```diff
@@ -593,7 +591,7 @@ var tcuImageCompare = framework.common.tcuImageCompare;
                             /** @type {number} */ var f1 = 0.5 + (xf - yf) * 0.5;
 
                             /** @type {Array<number>} */ var f = es3fFragmentOutputTests.swizzleVec([f0, f1, 1.0 - f0, 1.0 - f1], curInVec); // Vec4
-                            c = deMath.multiply(deMath.add(minVal, deMath.subtract(maxVal, minVal)), f); // Vec4
+                            c = deMath.add(minVal, deMath.multiply(deMath.subtract(maxVal, minVal), f)); // Vec4
```
Comment (qkmiao) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fFragmentOutputTests.cpp#671, C++ code:
Vec4	c	= minVal + (maxVal-minVal)*f;

File: sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js [hunk 3]
```diff
@@ -875,7 +873,7 @@ var tcuImageCompare = framework.common.tcuImageCompare;
 
                     finalThreshold = tcuTextureUtil.select(
                                     deMath.max(formatThreshold, [precThreshold, precThreshold, precThreshold, precThreshold]),
-                                    [1, 1, 1, 1], // C++ version: UVec4(~0u) bitwise not, all bits in the integer will be flipped
+                                    [0xffffffff, 0xffffffff, 0xffffffff, 0xffffffff], // C++ version: UVec4(~0u) bitwise not, all bits in the integer will be flipped
```
Comment (qkmiao) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fFragmentOutputTests.cpp#921
~0 should be 0xffffffff.

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**